src/styles/books.css

```diff
@@ -10,15 +10,90 @@
   border-radius: 3px;
 }
 
+.vert {
+  width: 1px;
+  height: 5rem;
+  background-color: #d3d3d3;
+}
+
 ul {
   display: flex;
   flex-direction: column;
   gap: 0.8rem;
 }
 
+.curr {
+  font-weight: 100;
+}
+
+.shortline {
+  color: #b8b8b8;
+  font-weight: 100;
+}
+
+.percentcontainer {
+  display: flex;
+  justify-content: center;
+}
+
+.progresssect {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] I noticed that you have multiple repeated CSS properties across your stylesheets. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/78251927/230717659-c3576a60-22ce-4f00-9f3e-0c9ee43bb4e2.png)

In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}

.flex-column {
  flex-direction: column
}
```
And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
<div class="flex flex-column"> This is a column-flexed container </div>
```